monkey/monkey_island/cc/services/edge.py

```diff
@@ -155,7 +155,7 @@ def get_edge_label(edge):
             else:
                 to_label = NodeService.get_node_label(NodeService.get_node_by_id(to_id))
 
-        RIGHT_ARROW = u"\u2192"
+        RIGHT_ARROW = "\\u2192"
```

Comment:
Need to make sure this is still working. Also, I think it might be preferable to just write ➡.